client/tests/doubles/client_api_stub.py

```diff
@@ -0,0 +1,23 @@
+from fenix import Event
+
+class ClientAPIStub:
+
+    def __init__(self, queue):
+        self._sent = None
+        self._queue = queue
+
+    def set_messages(self, messages):
+        for msg in messages:
+            self._queue.put((Event.MSG_RECV, msg))
+
+    def fail_connect(self):
+        self._queue.put((Event.CONN_FAIL, None))
+
+    def send(self, msg):
+        self._sent = msg
```

Comment:
Am I missing the part where this is made to be a `Event.MSG_SEND` or does it not exist?

Reply:
Either way I feel it would be better to directly put it in the queue

Reply:
It's more convenient for testing to use a simple variable than to get it off a queue.

Reply:
Correct.